packages/docs/base/elements.md

```diff
@@ -116,6 +115,64 @@ Many screen readers do not let users know of the presence of `del`. To fix this,
 ```
 </figure>
 
+## details <a name="details"></a>
+> The HTML Details Element (`<details>`) creates a disclosure widget in which information is visible only when the widget is toggled into an "open" state. A summary or label can be provided using the `<summary>` element. [...] If the first child of the `<details>` element is a `<summary>`, the contents of the `<summary>` element are used as the label for the disclosure widget.  - <cite><a href='https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details'>MDN</a></cite>
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
+    <details>
+      <summary>What is Okta?</summary>
```

Comment:
@arnoldsandoval-okta `summary` has the `button` role by default, so I'm treating a11y requirements as if it were `button`. that legit?

@joejezowski-okta because of above, I'm currently applying the button/input focus style here.

Reply:
Yeah you're right, this should be fine as is.